### ISSUE TITLE
Wrong hotkey mapping in TCnInputHelperForm (Tokyo)

### ISSUE THREAD

**Issue by CodehunterWorks** [cnpack/cnwizards#96]
Hello,

in TCnInputHelperForm at the Input Helper Tab, the "Shortcut for ..." Edits does wrong recognize pressed SHIFT (shows as ALT) and ALTGR (shows as CTRL+ALT)

Further, if i map the Input Helper Popup to ALT+SPACE, the normal IDE Popup shows althrough the normal Default-IDE-Shortcut for this is CTRL+SPACE. Both shows the normal IDE Popup, not the Cn Input Helper Popup.

Greets
Cody

**Comment by baka0815** [comment]
I can confirm at least the display problem (current CnWizards with Delphi 10.3.1 Rio).
Holding just SHIFT displays "ALT", holding STRG+ SHIFT or ALT+SHIFT (or all three) works as expected.
![1](https://user-images.githubusercontent.com/15894566/57511655-0c9d1880-730a-11e9-8654-fef2c3ec8817.png)
![2](https://user-images.githubusercontent.com/15894566/57511658-0d35af00-730a-11e9-949b-a196eba64bc1.png)

**Comment by CodehunterWorks** [comment]
> Holding just SHIFT displays "ALT", holding STRG+ SHIFT or ALT+SHIFT (or all three) works as expected.

Nope, works NOT as expected: Holding CTRL+SHIFT, it looks like working fine. But finally the CTRL+SHIFT+SPACE will "mutate" to CTRL+SPACE and this is already in use by the IDE. The behavior is likely related to the SPACE keycode because CTRL+SHIFT+<any Letter A..Z 0..9> does work fine.

**Comment by shanzhashu** [comment]
It seems to be the default behavior about "Pressing Shift Show Alt" in THotkey Control. But we can change the "InvalidKeys" Property and remove hcShift to correct it.

Alt+Space is a new hotkey in CnWizards' Source Editor Enhancement for IDE code insight popup. It can  be disabled in "CnPack", "IDE Enhancements Settings", "Editor Enhancements" dialog(first page).

**Comment by CodehunterWorks** [comment]
Sounds like so. I suggest an Hint on KeyPress which tells the user that the hotkey is already assigned to (which) another CnPack feature and a yes/no option to occupy it.
